x-pack/plugins/security_solution/public/siem_migrations/rules/components/siem_migrations_no_items_message.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiEmptyPrompt, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import React from 'react';
+import { SecurityPageName } from '../../../../common';
+import { useGetSecuritySolutionLinkProps } from '../../../common/components/links';
+import * as i18n from './translations';
+
+const SiemMigrationsTableNoItemsMessageComponent = () => {
+  const getSecuritySolutionLinkProps = useGetSecuritySolutionLinkProps();
+  const { onClick: onClickLink } = getSecuritySolutionLinkProps({
+    deepLinkId: SecurityPageName.rules,
```

Comment:
Shouldn't we go to the onboarding page?
```suggestion
    deepLinkId: SecurityPageName.landing, path: 'siem_migrations'
```